components/director/pkg/scenario/directive_test.go

```diff
@@ -443,3 +491,26 @@ func (d *dummyResolver) SuccessResolve(_ context.Context) (res interface{}, err
 func mockedNextOutput() string {
 	return "nextOutput"
 }
+
+func mockedTx(shouldCommit bool) *persistenceautomock.PersistenceTx {
```

Comment:
Generally you can use:
`txGen := txtest.NewTransactionContextGenerator(testErr)`